lib/commands/compatibility.js

```diff
@@ -50,6 +50,12 @@ async function main() {
             process.exit(1);
         }
 
+        if (plugin.latestBinary.compatibleSoftwareVersions[0].major === 'Shopware 6') {
+            console.log(`The plugin ${plugin.name} is compatible with Shopware 6 and minimum and maximum `
+            + 'compatibilities are set via composer.json there. This is not implemented yet.');
```

Comment:
Please don't split strings passed as arguments like this.
```suggestion
            console.log(
                `The plugin ${plugin.name} is only compatible with Shopware 6, which requires Shopware version `
                + 'constraints to be defined in its composer.json file. This is not supported yet.'
            );
```